src/main/java/com/leinardi/pycharm/mypy/mpapi/MypyRunner.java

```diff
@@ -272,19 +268,15 @@ public static String detectSystemMypyPath() {
         if (filesToScan.isEmpty()) {
             return Collections.emptyList();
         }
-        boolean daemon = false;
 
         GeneralCommandLine cmd = new GeneralCommandLine(mypyPath);
```

Comment:
The daemon has its own executable called "dmypy" which is next to "mypy" in the same folder. If useDaemon is true we would need to change that here.